component/admin/controllers/icalrepeat.php

```diff
@@ -466,7 +467,9 @@ private function doSave(& $msg)
 
 		$data["UID"] = $jinput->get("uid", md5(uniqid(rand(), true)));
 
-		$data["X-EXTRAINFO"]    = $jinput->get("extra_info", "", 'RAW');
+        if ($params->get("allowraw", 0)) {
+            $data["X-EXTRAINFO"] = $jinput->get("extra_info", "", 'RAW');
+        }
```

Comment:
Not quite - its the 'RAW' bit that needs the wrapper

e.g. 

`
if ($params->get("allowraw", 0)) {
            $data["X-EXTRAINFO"] = $jinput->get("extra_info", "", 'RAW');
        }
else {
            $data["X-EXTRAINFO"] = $jinput->get("extra_info", "");
}
`